decentraland/unity-renderer
# fix: add wait for loading screen on teleport

## What does this PR change?

Fixes the first case of https://github.com/decentraland/unity-renderer/issues/1615. Adds a new interaction for kernel, in which kernel waits for the loading screen to be visible before taking any action on the teleport.

Kernel branch: https://github.com/decentraland/kernel/pull/562


## How to test the changes?

TEST USING https://play.decentraland.zone/?renderer-branch=fix/Add-Fade-Function-For-Kernel&kernel-branch=fix/added-wait-function-for-loading

1. Start with a fresh copy of DCL
2. Accept terms and conditions
3. Check that the terms and condition canvas stays on until the loading screen fully fades in
4. Start the tutorial normally
5. Its also a good opportunity to confirm that the loading screen teleports work correctly for all the cases of https://github.com/decentraland/unity-renderer/issues/1615. For all the following cases, there should be no flash after teleporting.

Teleport from chat
Teleport from the Explore panel
Teleport from map
Teleport to any event from the Jump In button present on Genesis Plaza

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
